.github/workflows/helm-release.yml

```diff
@@ -0,0 +1,52 @@
+name: helm releaser
+
+on:
+  push:
+    branches: [master, main, force_test, release-*, staging, trying]
+    tags: ['*']
+  pull_request:
+    branches: ['**']
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.7.1
+
+      - name: Download latest index.yml
+        run: |
+          mkdir -p chart_release/
+          wget https://charts.timescale.com/index.yaml -O chart_release/index_old.yaml
+
+      - name: Create package
+        run: |
+          mkdir -p chart_release/packaged
+          helm package deploy/helm-chart -d chart_release/packaged
+          helm repo index chart_release/packaged --merge=chart_release/index_old.yaml  --url http://charts.timescale.com
+          # debug to be deleted
+          ls chart_release/packaged
+          cat chart_release/packaged/index.yaml
```

Comment:
Why not use https://github.com/hypnoglow/helm-s3 and then `helm s3 push` (especially since this tool is advertised by AWS themselves - https://docs.aws.amazon.com/prescriptive-guidance/latest/patterns/set-up-a-helm-v3-chart-repository-in-amazon-s3.html)?